### PR TITLE
Add TravisCI integration

### PR DESCRIPTION
**Pull request by sprsquish**

### CODE REVIEW

File: tests/test_case.py [hunk 1]
```diff
@@ -6,11 +6,11 @@
 from faunadb.errors import NotFound
 from faunadb.objects import Ref
 
-_FAUNA_ROOT_KEY = environ["FAUNA_ROOT_KEY"]
+_FAUNA_ROOT_KEY = environ.get("FAUNA_ROOT_KEY") or "secret"
```
Comment (ashfire908) [inline tests/test_case.py]: Should be able to use `.get("FAUNA_ROOT_KEY", "secret")` here

**Comment by sprsquish** [comment]
This now works with the core-nightly. Woot

File: tests/test_case.py [hunk 2]
```diff
@@ -6,11 +6,11 @@
 from faunadb.errors import NotFound
 from faunadb.objects import Ref
 
-_FAUNA_ROOT_KEY = environ["FAUNA_ROOT_KEY"]
+_FAUNA_ROOT_KEY = environ.get("FAUNA_ROOT_KEY", "secret")
```
Comment (ashfire908) [inline tests/test_case.py]: Actually thinking about this more, setting all these like this forces any end users who want to run tests unable to do so without overriding all the stuff we do here. Can we just revert these changes and configure the travis env to set these?

Comment (sprsquish) [inline tests/test_case.py]: I don't understand. They'll have to set all this stuff anyway if their environment is different.

**Comment by ashfire908** [comment]
Please rebase on master. You can drop the tests and readme fixes, and the dependencies/method of testing has changed. I'm not sure how you'll have to setup/call tests now.

**Comment by ashfire908** [comment]
Rebase on master and run test via setup.py.

**Comment by ashfire908** [comment]
What version of python does this run as?

**Comment by sprsquish** [comment]
I'm going to assume it's 2. I might be able to hack my way to 3 as well

**Comment by ashfire908** [comment]
@andy-faunadb to review

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+language: python
+python:
+  - "2.7"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+env:
+  global:
+    - secure: "F9dZyPKN1TGHGKi+VgFFcZxW2i2oN1lZC9D6U7uja/49GzszUF99CNhlvX9x/gRXatRdTEmPl/1YYJIgcQXiWuio33CPKjAab9PJEiXXP1Eo7izVAPsCXBh1+vsomGuSqJCel679jnIIACLEIj0PW2LRUycSPGObX424C9KxDHMG/pOa/+LWwHzY1czh33jbL4gEnSR2/nZqePEHv9XmmnoPGeWcQaXfvUcvWj3FWtaefP4xFXsBLkgfB2Q15jeIuqVx0z9NECBJdYXJktpJRRnkwT8LN5+gFa02MvQ8SUT6P6AlJ5Vg2eexO6C4YfCudML0M0Zh6spE3zCfpCWIXwvDd8zTOmDr3r3CP1NhMr0UQ1dcREEbXb+OKV9AlHKkmwFscgLHvlk3xlz3j9HUuEopQEFyUbYWeViq6OHnv862LWxHSrGd/vN2sY/xZoj9NfV3l5vFP0ECtfvvCf9AGeaJ/f4eKDOSu2GUrDtD5a6cx+ZVrwFUWYYn1JYY2l9jcG7hWcYwdB39yY5zw25Y+ATa2p2r8pF57iFS8+LlrzF3d6yOKGj66liRA1QuLxf8ihBYTtGM8csVyrCe0pt3MyYdQ5xMGNuj9poAiyJx0yonM5y4ShYXoEw2X7ds8tIt+C738EQQ/kzdpPXEeRVzimQsPDPYVXyl+yuZdLQBqU0="
+    - FAUNA_DOMAIN=rest.faunadb.com
```
Comment (andy-faunadb) [inline .travis.yml]: These are the Client default values, so we shouldn't need them.

Comment (sprsquish) [inline .travis.yml]: I'd like to make it explicit. This makes it obvious what's happening on Travis and ensures that if the defaults ever change, Travis wont.

**Comment by andy-faunadb** [comment]
LGTM
